index.js

```diff
@@ -289,6 +291,10 @@ function normalizeArguments(url, opts) {
 		opts
 	);
 
+	if (!opts.decompress) {
+		opts.encoding = null;
+	}
```

Comment:
*facepalm*. Of course, sorry.

What's the best solution here?

Copy/paste:

```js
if (['gzip', 'deflate'].indexOf(res.headers['content-encoding']) === -1)
```

straight from `decompress-response`?

Or maybe publish it separately as `responseIsGzipped()` and require in both Got/`decompress-response` so they're both in sync?